ffi/capi/js/examples/tinywasm/Makefile

```diff
@@ -25,14 +25,15 @@ crate/target/wasm32-unknown-unknown/release/crate.wasm: FORCE
 	rustup toolchain install ${ICU4X_NIGHTLY_TOOLCHAIN}
 	rustup component add rust-src --toolchain ${ICU4X_NIGHTLY_TOOLCHAIN}
 	# Build the WASM library
+	cd crate && \
 	RUSTFLAGS="-Cpanic=abort -Copt-level=s -C link-arg=-zstack-size=${WASM_STACK_SIZE} -Clinker-plugin-lto -Ccodegen-units=1 -C linker=${BASEDIR}/ld.py -C linker-flavor=wasm-ld -Clto -Cembed-bitcode" \
-	cd crate && cargo +${ICU4X_NIGHTLY_TOOLCHAIN} build \
+	cargo +${ICU4X_NIGHTLY_TOOLCHAIN} build \
```

Comment:
Issue: I think this might now put the weird flags in the top level target directory, no?

Reply:
No, this was broken since my cleanup PR, the RUSTFLAGS weren't set for the cargo invocation, only for the cd